web/gds-admin-ui/src/layouts/Topbar.js

```diff
@@ -25,13 +29,22 @@ type TopbarProps = {
     topbarDark?: boolean,
 };
 
+const api = new APICore();
+
 const Topbar = ({ hideLogo, navCssClasses, openLeftMenuCallBack, topbarDark }: TopbarProps): React$Element<any> => {
     const [isopen, setIsopen] = useState(false);
+    const dispatch = useDispatch()
     const { user, loading } = useSelector(state => ({
         user: state.Auth.user,
         loading: state.Auth.loading
     }))
 
+    React.useEffect(() => {
+        const { access_token } = api.getLoggedInUser()
+        const decodedToken = access_token && jwtDecode(access_token)
+        dispatch(loginUserSuccess(AuthActionTypes.LOGIN_USER_SUCCESS, decodedToken))
+    }, [dispatch])
```

Comment:
So this means that whenever the `Topbar` is rendered, the `getLoggedInUser` function may do reauthentication with the refresh token, so you're redispatching the successfully logged in user?